com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -109,7 +109,10 @@ def addOperator(self, kind, function=None, name=None, params=None, sl=None):
 
         if name is None:
             name = self._requested_name(None,action="Op", func = function)
-
+        
```

Comment:
Without this, SPL compilation fails when there are two operators of the same name.